packages/@uppy/transloadit/src/index.ts

```diff
@@ -877,84 +792,39 @@ export default class Transloadit<
   }
 
   #prepareUpload = async (fileIDs: string[], uploadID: string) => {
```

Comment:
uploadID is unused now
```suggestion
  #prepareUpload = async (fileIDs: string[]) => {
```